Source/WebInspectorUI/UserInterface/Views/ResourceClusterContentView.js

```diff
@@ -152,11 +152,11 @@ WI.ResourceClusterContentView = class ResourceClusterContentView extends WI.Clus
         if (this._responseContentView)
             return this._responseContentView;
 
-        this._responseContentView = this._contentViewForResourceType(this._resource.type);
+        let typeFromMIMEType = WI.Resource.typeFromMIMEType(this._resource.mimeType);
+        this._responseContentView = this._contentViewForResourceType(this._resource.type, typeFromMIMEType);
```

Comment:
this is kinda odd IMO :/

we're trying to find a `WI.ResourceContentView` that matches the `this._resource.type`, but also providing the `typeFromMIMEType` in case we want to use it as an override in other scenarios?

i feel like it'd be more clear/clean to maybe do the MIME attempt first (line 160) and then fall back to the resource type

alternatively, maybe we should adjust `WI.Resource.resolveType` to do this override instead?